macros/fivetran_log_lookback.sql

```diff
@@ -0,0 +1,15 @@
+{% macro fivetran_log_lookback(from_date, datepart='day', interval=7, default_start_date='2010-01-01') %}
+
+{{ adapter.dispatch('fivetran_log_lookback', 'fivetran_log') (from_date, datepart='day', interval=7, default_start_date='2010-01-01') }}
+
+{%- endmacro %}
+
+{% macro default__fivetran_log_lookback(from_date, datepart='day', interval=7, default_start_date='2010-01-01')  %}
+
+    coalesce(
```

Comment:
True. I trialed removing this and it seems to compile and run fine. However for now I will leave this pending discussion on Tuesday.

Reply:
Thanks, I just wanted to make sure there likely wouldn't be a scenario where this field would actually be `null`. I don't believe including it will hurt so I am comfortable leaving it in.